### DISCUSSION TITLE
Unable to open app after update

### DISCUSSION THREAD

**Discussion by PurkiStreamTeam** [GetPublii/Publii#1637] [Help] [answered]
Hello! I just recently download the latest version of publii- 0.46.1 (build 16918) on windows to update. But after the installation i got this error and it prevents me to open the app, Is there any solution to this?

![image](https://github.com/user-attachments/assets/c89350c3-c823-4cb3-a255-08057c32fcef)

**Comment by dziudek** [comment, marked as answer]
Hi @PurkiStreamTeam - most probably something goes wrong while removing simple theme files - maybe with permissions. Please try to remove manually C/Users/User/Documents/Publii/themes/simple folder and try again
